source/CardGames.py

```diff
@@ -128,76 +160,146 @@ def printCards(self, cards: "list[Card]", showFront: bool, printShort: bool = Tr
-        if guess_all == 'y' or guess_all == 'yes':
-            while(True):
-                guess = input("Enter locations separated only by a comma (1,2,3...)").split(',')
-                """
-                if len_list < placed_locations:
-                    print("Must guess all locations")
-                    guess = input("Enter locations separated only by a comma (1,2,3...)").split(',')
-                    continue
-                """
-                # Check that input is a valid list of ints
-                bad = False
-                for ele in guess:
-                    try:
-                        ele = int(ele)
-                    except:
-                        TypeError
-                        print("Invalid input, try again")
-                        bad = True
-                        break
-                if bad:
-                    continue
-                break
-            break
-        elif guess_all == 'n' or guess_all == 'no':
-            while True:
-                try:
-                    guess = int(input("Enter fugitive location..."))
-                except:
-                    ValueError
-                    print("Invalid input, try again")
-                    continue
-                break
-            break
-        else:
-            print("Invalid input, try again")
-
-    #marshall.add_guess(guess)
-    print("Guesses added")
-    """
-    if check_location(marshall_current_idx, guess):
-        print("Correct location!")
-        reveal_location(marshall_current_idx)
-    else:
-        print("Incorrect guess.")
-    """
+class GameSetup():
+  def __init__(self):
+        self.escape_card, self.HighRangeDeck, self.MidRangeDeck, self.LowRangeDeck, self.starting_cards = split_card()
+        self.cards_in_play = []
+        self.marshall = Player('joe')
+
+  def display_board_general(self):
+    sprints = [stack for stack in self.cards_in_play if type(stack) == list]
+    has_sprints = False
+    for idx in range(6):
+        for stack in self.cards_in_play:     
+          if type(stack) == list:
+            has_sprints = True
+            card = stack[0]
+          else:
+            card = stack
+          image = card.image[idx] if card.revealed else card.cardBack[idx]
+          print(image, end="")
+        print()
+    if has_sprints:
+      spacing = len([s for s in self.cards_in_play if type(s) != list])
+      
+      for i in range(1, max([len(stack) for stack in sprints])):
+        for idx in range(2, 6):
+          print(' ' * 7 * spacing, end="")
+          for stack in sprints:  
+            if len(stack) > i:
+              if idx == 5:
+                print('|_____|', end="")
+              else:
+                card = stack[i]
+                image = card.image[idx]
+                print(image, end="")
+            else:
+              print(' ' * 7, end="")
+          print()
+
+  def check_location(self, guess, marshall_current_idx):
```

Comment:
Hey Sean, is this function always returning true? Is there some shorthand I am missing?

Reply:
Its just a filler function since we haven't implemented the function it needs. I'll put a comment by that.